src/components/spark/spark.component.less

```diff
@@ -71,4 +175,12 @@ ux-spark {
     .ux-spark-multi-value {
         display: flex;
     }
+}
+
+.ux-spark-theme(@color) {
```

Comment:
Try and add the class name to mixin to simplify this further, so you could pass in, `.ux-spark-theme('primary', @primary)` and it would generate:

```less
&.ux-spark-theme-primary {
    background-color: fade(@primary, 20%);

    .ux-spark-bar {
        background-color: @primary;
    }
}
```